Sample_v3/Extensions/Channel+MembersInfo.swift

```diff
@@ -5,7 +5,7 @@
 import StreamChatClient
 
 func createMemberInfoString(for channel: ChatChannel) -> String {
-    "\(channel.members.count) members, \(channel.members.filter(\.isOnline).count) online"
+    "\(channel.memberCount) members, \(channel.watcherCount) online"
```

Comment:
But the original implementation was also incorrect because the `members` doesn't contain all members. This seems to be _less_ wrong than the original implementation. For example, in LLC v3 we all locally known channels are watched by default.